### PR TITLE
Doc/174 doc presentation and script 12

### PR DESCRIPTION
**Pull request by AntonioRodriguezRuiz**
# Description
Please include a summary of the changes and the related issue. Please also include relevant motivation and context. 

# Checklist:
- [x] I have performed a self-review of my code running `npm run build` and `npm run serve` after pushing the code.
- [x] I have checked that there is no fragile information published unintentionally.

# Screenshot:

### CODE REVIEW

File: docs/Presentaciones/02-04-2024.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+# Guión para la presentación de evaluación del Sprint 2
+
+| Guión preliminar por | Presentador       | Tiempo  |
+| -------------------- | ----------------- | ------  |
+
+## Killer Opener + Elevator Pitch
+
```
Comment (alvarobernal2412) [inline docs/Presentaciones/02-04-2024.md:7]: The killer opener is missing

File: docs/Presentaciones/02-04-2024.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+la correcta aplicación del sistema de avisos.
+
+**FINALIZA OPCIONAL**
+
+A lo largo del proyecto este cumpolimiento ha sido excelente tanto a nivel general como indivual, quitando alguna semana puntual en la que algún miembro hubiera recibido algun aviso por no cumplir con
+la clausula relaciuonada con el cumplimiento de tareas o justificación de retrasos.
+
+Especialmente este Sprint ha sido excepcional sin ninguna incidencia.
+
+_Destacar a los mejores miembros del equipo y a los que más han trabajado_
```
Comment (alvarobernal2412) [inline docs/Presentaciones/02-04-2024.md:150]: The narration of the prototype is missing, although I know that you are in the process of editing the video and that is why this section is not yet here.

File: docs/Presentaciones/02-04-2024.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+## ROI
+
+Y bien, 91.000€ no es poco dinero, nos hace falta estar seguros de que vamos a recuperar esta inversión. Por ello hemos hecho un análisis tratando de predecir el crecimiento de nuestra base de clientes y
+el retorno de la inversión en situaciones optimistas, pesimistas y más probables.
+
+Como veis en la gráfica, los costes son un poco mas grandes al principio, en el periodo de desarrollo, pero conforme entramos en producción bajan, a la vez que aumentan los ingresos. Segun hemos calculado
+nuestro servicio seria generaría mas de lo que cuesta a partir de los 30 meses aproximadamente tanto en el escenario optimista como en el más probable, y en el pesimista, que asume un crecimiento
+practicamente nulo de clientes, no parece que vayamos a recuperar la inversión.
+
+## Seguimiento de costes + burndown
```
Comment (alvarobernal2412) [inline docs/Presentaciones/02-04-2024.md:115]: An enriching analysis

**Review by alvarobernal2412 — CHANGES_REQUESTED** [review]
Well done! Only a couple of details need to be finalised

File: docs/Presentaciones/02-04-2024.md (hunk 1 above)
Comment (alvarobernal2412) [reply docs/Presentaciones/02-04-2024.md:7]: Sorry for my oversight, I was reviewing it without updating the local changes.

**Comment by AntonioRodriguezRuiz** [comment]
@alvarobernal2412 it should all be fixed now

**Review by alvarobernal2412 — APPROVED** [review]
Nice job!

**Pull request merged**